applications/RomApplication/costum_strategies/rom_builder_and_solver.h

```diff
@@ -0,0 +1,682 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt

```

Comment:
Header format

Reply:
@rubenzorrilla I have gone through the files to check the indentation formatting